jenkins/scripts/integration_test.sh

```diff
@@ -43,6 +43,12 @@ DEFAULT_HOSTS_MEMORY="8192"
 VM_TIMELABEL="${VM_TIMELABEL:-$(date '+%Y%m%d%H%M%S')}"
 TEST_EXECUTER_VM_NAME="${TEST_EXECUTER_VM_NAME:-ci-test-vm-${VM_TIMELABEL}}"
 TEST_EXECUTER_PORT_NAME="${TEST_EXECUTER_PORT_NAME:-${TEST_EXECUTER_VM_NAME}-int-port}"
+
+if [ "${TESTS_FOR}" == "feature_tests" ]
+then
+  TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-8C-32GB-300GB}"
```

Comment:
Was not it 500GB disc space for proper testing?

Reply:
I think 300 GB is enough. But if turns out to be not enough during the tests, then we will increase it.